src/runtime/index.ts

```diff
@@ -177,15 +171,14 @@ export function runtime(defaults: RequestOpts) {
         }
       });
 
+      // Remove the Content-Type header because it should be automatically set
+      // by the Fetch API.
+      delete headers?.["Content-Type"];
```

Comment:
I think will also delete "Content-Type" headers intentionally set by the user. Which is behavior I wouldn't expect.

Reply:
My understanding is that when the request body is `FormData`, you must not specify anything in the Content-Type header. Therefore, if the Content-Type header is specified at this point, it would be a error on the user's side.
That said, I don't think we need to go out of our way to correct such mistakes, so I'll follow your approach.